.github/workflows/pr-auto-assign.yml

```
@@ -8,7 +8,7 @@ permissions:
 
 jobs:
   add-reviews:
-    if: ${{ join(github.event.pull_request.requested_reviewers.*.login, ',') == '' }}
+    if: ${{ join(github.event.pull_request.requested_reviewers.*.login, ',') == '' && github.event.pull_request.draft != "True" }}
```

```suggestion
    if: ${{ join(github.event.pull_request.requested_reviewers.*.login, ',') == '' && github.event.pull_request.draft == false }}
```
https://stackoverflow.com/a/68349262 :shrug: